icons/square/48/gimagereader.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="48" height="48" version="1.1" viewBox="0 0 48 48.000001" xmlns="http://www.w3.org/2000/svg">
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This has that weird header with width and heigh attributes again. Maybe not something we need to worry about for individual icons, but we can replace them all en mass with one which we know works.